.github/pages/snippets.md

```diff
@@ -0,0 +1,694 @@
+# Jina Code Snippets
+
+These code snippets cover all aspects of Jina's functionality. To run a snippet in a Jupyter Notebook, just click the "run" button next to the snippet.
```

Comment:
I wouldn't claim they cover all. They are a short and easy introduction to Jina functionality and design framework.

Reply:
Fixed